.github/workflows/cpp.yml

```diff
@@ -186,9 +186,9 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - windows-latest
+          - windows-2019
         include:
-          - os: windows-latest
+          - os: windows-2019
```

Comment:
@kszucs I'm pretty sure our builds fail if run on Windows 2022.  They specify the 2019 generator and it can't be found.  Do you want to update the generator?  Or downgrade the GHA runner?

Reply:
I'd rather downgrade the GHA runner for now as an older generator is more likely to hit limitations.

Reply:
I'd also prefer to pin the older version.